### ISSUE TITLE
[0018] Rota de "Minha credencial"

### ISSUE THREAD

**Issue by FilipePfluck** [diego3g/rsxp-2023#69]
Vou começar a trabalhar nessa task

**Comment by diego3g** [comment]
Mesma coisa que acabei de comentar na issue #68, vale a pena esperar um pouco mais pra tocar essas tasks porque dependem bastante da parte de autenticação sendo feita na #61.

**Comment by FilipePfluck** [comment]
@diego3g sim, faz sentido. Eu tava pensando em baixar o projeto e dar uma olhada, e acompanhar as outras issues pra daí começar a mexer nessa aqui. Eu só abri a issue mesmo por que sobraram poucas tasks e eu queria trabalhar em alguma coisa 🤠. Prefere que eu já comece trabalhando em outra issue e deixe essa aqui em aberto mesmo?

**Comment by FilipePfluck** [comment]
fala @diego3g agora que o #61 foi mergeado eu já posso começar a trabalhar nessa aqui, certo?
